packages/schema/src/chip/index.ts

```diff
@@ -0,0 +1,32 @@
+import { ColorTokens } from '../colors';
+import { CornerRadiusTokens } from '../cornerRadius';
+import { IconProps } from '../icon';
+import { SpaceTypeTokens } from '../space';
+import { FontSizeTokens } from '../typography';
+
+export type ChipProps = {
+	/**
+	 * Button Label default Button
+	 */
+	label?: String;
+	/**
+	 * Default Icon null
+	 */
+	icon?: IconProps;
+	/**
+	Default iconAlignment - right
+    */
+	iconAlignment?: 'left' | 'right';
+	bgColor?: ColorTokens;
+	labelColor?: ColorTokens;
+	fontSize?: FontSizeTokens;
+	borderRadius?: CornerRadiusTokens;
+	padding?: {
```

Comment:
can we also consider to add separate padding schema. this is repeating in button as well